charts/ccd-definition-store-api/values.aat.template.yaml

```diff
@@ -2,4 +2,14 @@ java:
   # Don't modify below here
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
+  environment:
+    DEFINITION_STORE_DB_HOST: "{{ .Release.Name }}-postgresql"
+    DEFINITION_STORE_DB_NAME: "{{ .Values.postgresql.postgresqlDatabase}}"
+    DEFINITION_STORE_DB_USERNAME: "{{ .Values.postgresql.postgresqlUsername}}"
+    DEFINITION_STORE_DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
+    DEFINITION_STORE_DB_OPTIONS: "?sslmode=require"
```

Comment:
this is not correct, why we are switching from azure to aks db?

Reply:
I agree  the host to be used is the one in values .  ccd-definition-store-api-postgres-db-{{ .Values.global.environment }}.postgres.database.azure.com .